js/ui/components/clubhouse.js

```diff
@@ -369,13 +375,17 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
     // Override this method because we don't want the button
     // horizontally expanded:
     addButton(button, callback) {
-        if (button.label === '>') {
+        if (Object.keys(CLUBHOUSE_ICONS).includes(button.label)) {
```

Comment:
I can see the previous code already worked by checking if the button label was a particular string. Now that we have three possibilities for icon buttons, is it now time to find a more robust way of specificing it? I would try to avoid (undocumented?) magic values. In particular this seems sensitive to typos and accidental extra whitespace.

Reply:
Indeed this is not robust at all. Unfortunately `Gio.Notification` doesn't have a way to pass icons to buttons. I'm thinking if there is a more robust way to do it. If not, would documenting this work? The `>` is usually added by the Clubhouse code, but the thumbs-up, thumbs-down comes directly from the dialogues spreadsheet, sanitized on import but always prone to typos.

Reply:
We discussed on Slack that since these strings come from the dialogue spreadsheet, having the magic value is probably the best option, better than having separate columns for text and image for each button.